### DISCUSSION TITLE
Visual code

### DISCUSSION THREAD

**Discussion by David230798** [balta-io/2801#17] [Perguntas e Respostas] [answered]
Boa tarde professor, como eu não consegui  solução para minha ultima publicação sobre o erro de compilação, resetei meu pc, voltei as aulas sobre uso do VISUAL, instalei os sdks e tudo mais, porém ao digitar shell commands depois de ctrl + shift + p não aparece nada
![visual](https://user-images.githubusercontent.com/92813397/137953611-fe52156b-0fbd-4172-81f7-0149093d2f83.png)

**Comment by David230798** [comment]
Encontrei também um erro de formatação, não sei se já viu, mas só pra corrigir...

Divisão e multiplicação tá invertido

![erro curso](https://user-images.githubusercontent.com/92813397/137965996-4f48f69b-f5bd-4a41-9191-ff997172d47a.png)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia @David230798 como vai?
Em qual aula você está e o que está tentando fazer? Assim posso te ajudar melhor 💜
